packages/material/scss/input/_theme.scss

```
@@ -48,7 +48,7 @@
 
     // Floating label
     .k-textbox-container {
-        vertical-align: top;
+        vertical-align: 0;
```

Please, use `baseline` here, because the `vertical-align` is a computed property.
```suggestion
        vertical-align: baseline;
```